test/jdk/javax/management/remote/mandatory/notif/policy.negative

```diff
@@ -4,4 +4,5 @@ grant {
     permission javax.management.MBeanPermission "*", "invoke";
     permission javax.management.MBeanPermission "[domain:type=NB,name=2]", "addNotificationListener";
     permission javax.management.MBeanPermission "*", "removeNotificationListener";
+    permission javax.security.auth.AuthPermission "doAs";
```

Comment:
I suspect that this means a doPrivileged is missing somewhere. We should not require the application to posess new permissions.

Reply:
I think Daniel is right, can you remove this permission and paste in the debug output to see where this is happening?

Reply:
Hmm I may have fixed that since chaning the policy files, as I'm not seeing the problem without that AuthPermission any more.  Am just retesting everything before updating this...

Reply:
(Same with other policy files in which the permission was added of course)